index.html

```diff
@@ -9,18 +9,35 @@
         <script src="js/fruits_and_veggies_test.js"></script>
```

Comment:
Here is my banner for winning:

    <div ng-show="win" ng-class="{'has-success': win}">
        Congrats! You Won!
    </div>